partner_address_street3/hooks.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 Nicolas Bessi, Alexandre Fayolle, Camptocamp SA
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def post_init_hook(cr, registry):
+    query = """
+        UPDATE res_country
+        SET address_format = replace(
+        address_format,
+        E'%(street2)s\n',
+        E'%(street2)s\n%(street3)s\n'
+        )
+    """
+    cr.execute(query)
+
+
+def uninstall_hook(cr, registry):
+    query = """
+        UPDATE res_country
+        SET address_format = replace(
+        address_format,
+        E'%(street3)s\n',
```

Comment:
I think the `\n` part here should be optional. If any `%(street3)s` is found without a trailing `\n` it should get removed too.

Also please add docstrings to these hooks to make it clearer that you are adding/dropping street3 to the standard display format for addresses automatically.